Easydict/App/Localizable.xcstrings

```diff
@@ -1082,7 +1082,7 @@
         "zh-Hans" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "词源:"
+            "value" : "词源："
```

Comment:
This doesn't need to be modified, in fact, it's not being used at the moment. 

When rewriting with Swift later on, this part of the code can be removed.

Reply:
Maybe keep it for now?  We could delete it when once the label has been removed.

Reply:
Yes, keep it.